### ISSUE TITLE
Dictionary doesn't works in v1.2.1

### ISSUE THREAD

**Issue by paulcarroty** [johnfactotum/foliate#59]
```js
$ com.github.johnfactotum.Foliate 
Gjs-Message: 17:49:53.037: JS WARNING: [resource:///com/github/johnfactotum/Foliate/js/main.js 448]: reference to undefined property "Paginated"

(com.github.johnfactotum.Foliate:4843): Gjs-WARNING **: 17:49:53.037: JS ERROR: TypeError: this._buttons[x] is undefined

```

**Comment by johnfactotum** [comment]
Yes, sorry for this. The root of the problem has been fixed in 410224f3d32775069ecff331b069712d7678241b. In 1.2.0 the paginated layout was replaced by auto and single, so now it can't find a layout called "Paginated", and so everything breaks down.

As a workaround, switch the layout to 'Single' or 'Scrolled' then back to 'Auto', then restart the app, and it should work.

**Comment by paulcarroty** [comment]
@johnfactotum rebuild from master and it works, thanks. Great project, man, don't stop!
